pkg/codegen/hcl2/README.md

```diff
@@ -82,14 +83,17 @@ type without having to couple the type checker to how a resource's type is an
 object consisting of its output properties.
 
 This package's types mirror (and typically wrap) the types of the AST, adding
-semantic information such as types and resolved references. So for instance, a
-`model.LiteralValueExpression`, which corresponds to a literal value such as the
-string `"foo"` or the number `42`, contains:
+semantic information such as [datatypes](pcl-type-system) and resolved
```

Comment:
Trying to make the distinction between Go types and the PCL type system types, not sure what the best approach is.